lib/Sema/ImportResolution.cpp

```diff
@@ -623,8 +623,8 @@ void ImportResolver::addImplicitImports() {
   const ModuleDecl *moduleToInherit = nullptr;
   if (underlyingClangModule) {
     moduleToInherit = underlyingClangModule;
-    boundImports.push_back(
-        AttributedImport(ImportedModule(underlyingClangModule)));
+    boundImports.push_back(AttributedImport<ImportedModule>(
+        ImportedModule(underlyingClangModule)));
```

Comment:
Could also switch to `emplace_back`, I don't really feel too strongly either way though